constraint/config/crds/templates_v1alpha1_mutationtemplate.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
This code should be autogened via `make manifests`. Was this done by hand? This goes hand-in-hand w/kubebuilder